openssl/src/ssl/mod.rs

```diff
@@ -1814,6 +1815,13 @@ impl SslCipherRef {
             String::from_utf8(CStr::from_ptr(ptr as *const _).to_bytes().to_vec()).unwrap()
         }
     }
+
+    /// Returns the handshake digest of the cipher.
```

Comment:
We should note the normal things up here like the OpenSSL function and version requirements.